src/client/components/Status/Status.jsx

```diff
@@ -0,0 +1,22 @@
+import { useEffect, useState } from 'react';
+import styles from './status.module.css';
+
+export const Status = ({ text }) => {
+  const [isActive, setIsActive] = useState(false);
+
+  useEffect(() => {
+    let timer;
+
+    if (text) {
+      setIsActive(true);
+
+      timer = setInterval(() => setIsActive(false), 5000);
```

Comment:
Using `setInterval` here is not ideal for a one-time delay. Consider using `setTimeout` instead, as it better suits the purpose of hiding the status message after a fixed duration. This will also simplify the cleanup logic.